sdks/java/io/hbase/src/main/java/org/apache/beam/sdk/io/hbase/HBaseIO.java

```diff
@@ -168,68 +168,43 @@ public static Read read() {
      * @see HBaseIO
      */
     public static class Read extends PTransform<PBegin, PCollection<Result>> {
-        /**
-         * Returns a new {@link HBaseIO.Read} that will read from the HBase instance
-         * indicated by the given configuration.
-         */
+        /** Reads from the HBase instance indicated by the given configuration. */
```

Comment:
Just as a drive-by. Seemed not worth sending a separate PR - but the new documentation is more consistent with the rest of Beam and is more concise.